docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/sort/search-sorted-15.rst

```diff
@@ -0,0 +1,72 @@
+SearchSorted
+===============
+
+
+.. meta::
+  :description: Learn about SearchSorted - a sorting and maximization
+                operation, which requires two input tensors.
+
+
+**Versioned name**: *SearchSorted-15*
+
+**Category**: *Sorting and maximization*
+
+**Short description**: Find the indices from the innermost dimension of sorted_sequence such that, if the corresponding values in values were inserted before the indices, when sorted, the order of the corresponding innermost dimension within sorted_sequence would be preserved.
+
+**Detailed description**: `Reference <https://pytorch.org/docs/stable/generated/torch.searchsorted.html>`__.
```

Comment:
Please give our normal description for this operation and share examples for this operation. Just give a reference to PyTorch is not good since it can be changing so our operation is also automatically changing.

Reply:
1. What is a 'normal description for this operation'?

2. So I will remove the reference at all, since the only refs I have found are from pyTorch or Numpy, further more, I was explicitly told to do it exactly as in Pytorch, since fast path is needed foe this op.

Reply:
Let us have our own description and give examples.
What if they change description in PyTorch we will have to re-implement our functionality.

Reply:
Spec is rewritten - hope will be fine now.